subscription/upgrade.md

```diff
@@ -7,29 +7,29 @@ redirect_from:
 - /docker-hub/billing/upgrade/
 ---
 
-This page contains information on how to purchase a Docker paid subscription. It also contains information on how to upgrade your existing subscription and update your billing information.
+When you upgrade to a paid subscription, you immediately have access to all the features and entitlements available in your new chosen subscription. For detailed information on features available in each subscription, see [Docker Pricing](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"}.
 
-You can upgrade from Docker Personal to a Pro, Team, or a Business subscription from a Personal subscription. When you upgrade to a paid subscription, you can immediately use all the features and entitlements offered in your new subscription. For detailed information on features available in each subscription, see [Docker Pricing](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"}.
+## Upgrade from a free subscription to a paid subscription 
 
-## Upgrade to a Pro, Team, or Business subscription
+1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.
 
-The following section contains information on how to purchase a Docker  subscription for new customers. To upgrade from a Personal subscription to a Team or Business subscription, you must first convert your personal namespace to an organization namespace. To learn more, see [convert an account into an organization](../docker-hub/convert-account.md).
+2. Convert your personal namespace to an organization namespace. To learn more, see [convert an account into an organization](../docker-hub/convert-account.md).
```

Comment:
**issue:** we need to rework step 2 because it's not required.

1. We should use `namespace` only when referring to repositories. In our case, it should be `user account` and `organization account`.
2. Converting a user account to an organization account is an option if a customer whats to use keep using an account name (e.g. `netflix`) and upgrade it to Team or Business subscription.